### DISCUSSION TITLE
deprecations.log

### DISCUSSION THREAD

**Discussion by ptutty** [MasaCMS/MasaCMS#192] [Q&A]
We are about to replace Mura with Masa with in production (using a new dockerised env). I've noticed a new log file with Masa:

deprecations.log

When I tail it i get:
`Linenumber: 17: Template: /var/www/html/core/modules/v1/deprecation/model/handlers/handler.cfc: Function: ONLOGDEPRECATION
Linenumber: 1769: Template: /var/www/html/core/mura/plugin/pluginManager.cfc: Function: EXECUTESCRIPTS
Linenumber: 1307: Template: /var/www/html/core/mura/plugin/pluginManager.cfc: Function: ANNOUNCEEVENT
Linenumber: 401: Template: /var/www/html/core/mura/MasaScope.cfc: Function: ANNOUNCEEVENT
Linenumber: 86: Template: /var/www/html/core/mura/MuraScope.cfc: Function: INIT
Linenumber: 365: Template: /var/www/html/core/mura/queryParam.cfc: Function: GETCONTENTRENDERER
Linenumber: 220: Template: /var/www/html/core/mura/queryParam.cfc: Function: SETCRITERIA
Linenumber: 103: Template: /var/www/html/core/mura/queryParam.cfc: Function: INIT
Linenumber: 449: Template: /var/www/html/core/mura/content/feed/feedGateway.cfc: Function: GETFEED
Linenumber: 132: Template: /var/www/html/core/mura/content/feed/feedManager.cfc: Function: GETFEED"`

what is this log for? Should i be concerned? My site seems to work okay so far..

**Comment by guustnieuwenhuis** [comment]
Hi @ptutty,

Nice to hear you're migrating to Masa CMS!

You'll find the details on the deprecation log here:
https://docs.masacms.com/deprecations/deprecation_warnings/
